src/dockerfiles/connectors/test/start-snowflake.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+git clone https://github.com/aqueducthq/aqueduct.git
```

Comment:
@hsubbaraj-spiral This should cause the code to be fetched every single time the pod is run, so you don't need to repush a new image everytime the code changes.